summingbird-online/src/main/scala/com/twitter/summingbird/online/option/AllOpts.scala

```diff
@@ -98,8 +99,22 @@ trait SummerBuilder extends Serializable {
 
 /**
  * The SummerConstructor option, set this instead of CacheSize, AsyncPoolSize, etc.. to provide how to construct the aggregation for this bolt
+ * @see [[Summers]] for useful [[SummerConstructor]]s.
  */
-case class SummerConstructor(get: SummerBuilder)
+case class SummerConstructor(get: SummerConstructorSpec)
+
+trait SummerConstructorSpec {
```

Comment:
Why is it called SummerConstructorSpec, there's no SummerConstructor in definition.